### PR TITLE
Workspace: navigation update

### PR DESCRIPTION
**Pull request by gustavlrsn**
## Description
Updated navigation pattern behind the workspace feature flag:
- hamburger site menu drawer
- profile drawer with existing My Collectives/Events/Orgs list (to not disrupt existing user patterns too much)
- header only showing current page, with breadcrumbs for Collectives/Events

## Screenshots
image
image
image

### CODE REVIEW

File: pages/createOrganization.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class CreateOrganizationPage extends React.Component {
     }
 
     return (
-      <Page>
+      <Page title="Create Organization">
```
Comment (kewitz) [inline pages/createOrganization.js:28]: Intl.

File: pages/root-actions.js [hunk 2]
```diff
@@ -126,13 +126,8 @@ const RootActionsPage = () => {
...
   const showHiddenActions = Boolean(router.query.showHiddenActions);
   return (
-    <AuthenticatedPage disableSignup rootOnly>
-      <Container maxWidth="1000px" m="0 auto" mt={4} borderBottom="1px solid #e5e5e5">
-        <H1 textAlign="left" fontSize="32px" py={2} pl={2}>
-          Root actions
-        </H1>
-      </Container>
-      <Grid gridTemplateColumns={GRID_TEMPLATE_COLUMNS} maxWidth="1000px" m="0 auto" mb={5}>
+    <AuthenticatedPage disableSignup rootOnly navTitle="Root Actions">
```
Comment (kewitz) [inline pages/root-actions.js:129]: Intl. navTitle.

**Review by kewitz — COMMENTED** [review]
Looking good and self-contained. We should be able to merge this after all tests are passed.
Also, make sure to internationalize those strings, otherwise, they'll never be updated. 😂

File: pages/createOrganization.js (hunk 1 above)
Comment (gustavlrsn) [reply pages/createOrganization.js]: Removed this one, thinking the nav title is not necessary on certain pages that have a big title on the page itself

**Review by kewitz — APPROVED** [review]

**Pull request merged**